### PR TITLE
Sitemap

### PR DESCRIPTION
**Pull request by mccalluc**
towards #347. When we have automatic reingests, we will get automatic sitemaps.

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -7,3 +7,38 @@ Rails.application.load_tasks
...
+  # Solr field used to retrieve from a document the value for the priority element for a url
+  # sm.priority_field = nil
+
+  # Solr query sort parameter
+  # sm.sort = '_docid_ asc'
+  
+  # Solr request handler. This can be useful when your Solr configuration already has
+  # a filter query appended.
+  sm.qt = nil
+end
```
Comment (afred) [inline Rakefile]: Generally, you'd define rake tasks in `lib/tasks/whatever.rake` using syntax like...
```ruby
namespace :blacklight do
  desc "something awesome"
  task :something_awesome do
    # code here
  end
end
```
Is there a reason for putting it straight into the Rakefile?

Comment (mccalluc) [inline Rakefile]: This is what the generator gave me out of the box, and I didn't know that it was un-idiomatic. Worth fixing?

File: scripts/download_clean_ingest.rb [hunk 2]
```diff
@@ -173,10 +174,14 @@ def process()
...
       $LOG.info('Finished one big commit.')
     end
-
+    
     # TODO: Investigate whether optimization is worth it. Requires a lot of disk and time.
     # puts 'Ingest complete; Begin optimization...'
     # ingester.optimize
+    
+    $LOG.info('Starting sitemap generation...')
+    $LOG.info(`rake blacklight:sitemap`) # TODO: figure out the right way to do this.
```
Comment (afred) [inline scripts/download_clean_ingest.rb]: maybe tryi switching to

```ruby
Rake::Tasks['blacklight:sitemap'].invoke
```

File: Rakefile (hunk 1 above)
Comment (afred) [inline Rakefile]: i don't know that it's not idiomatic either. Can clean it up later if Rakefile gets too messy.

File: scripts/download_clean_ingest.rb (hunk 2 above)
Comment (mccalluc) [inline scripts/download_clean_ingest.rb]: I don't think I can do that unless I pull in a lot more of rake, and I'm not prepared to get in to a big refactoring right now. (At the very least, there will be conflicts with the PR that just came in.) I've filed #357 for the follow-up.

**Pull request merged**